src/applications/check-in/pre-check-in/app-entry.jsx

```diff
@@ -8,6 +10,11 @@ import reducer from '../reducers';
 import manifest from './manifest.json';
 
 import '../utils/i18n/i18n';
+import '../utils/defineWebComponents';
+
+if (!environment.isProduction()) {
+  import('platform/site-wide/sass/minimal.scss');
```

Comment:
I've made this a conditional import so that we don't have the site-wide stylesheet _and_ this minimal stylesheet in production. Once the pre-check-in app (and its surrounding context like the site header and footer) have been confirmed to look fine in dev & staging, we can [remove the prod condition in `content-build`](https://github.com/department-of-veterans-affairs/content-build/pull/1358#discussion_r1021922214) and [undo this conditional import as mentioned in the commit](https://github.com/department-of-veterans-affairs/vets-website/pull/22667/commits/017fb54a869467bb1b569e331ccd3def64239596).